src/arena.c

```diff
@@ -191,16 +198,25 @@ arena_stats_merge(tsdn_t *tsdn, arena_t *arena, unsigned *nthreads,
 	    &arena->stats.mapped);
 	astats->retained += (extents_npages_get(&arena->extents_retained) <<
 	    LG_PAGE);
-	astats->npurge += arena_stats_read_u64(tsdn, &arena->stats,
-	    &arena->stats.npurge);
-	astats->nmadvise += arena_stats_read_u64(tsdn, &arena->stats,
-	    &arena->stats.nmadvise);
-	astats->purged += arena_stats_read_u64(tsdn, &arena->stats,
-	    &arena->stats.purged);
+
+	astats->decay_dirty.npurge += arena_stats_read_u64(tsdn, &arena->stats,
+	    &arena->decay_dirty.stats.npurge);
+	astats->decay_dirty.nmadvise += arena_stats_read_u64(tsdn,
+	    &arena->stats, &arena->decay_dirty.stats.nmadvise);
+	astats->decay_dirty.purged += arena_stats_read_u64(tsdn, &arena->stats,
+	    &arena->decay_dirty.stats.purged);
+
+	astats->decay_muzzy.npurge += arena_stats_read_u64(tsdn, &arena->stats,
+	    &arena->decay_muzzy.stats.npurge);
+	astats->decay_muzzy.nmadvise += arena_stats_read_u64(tsdn,
+	    &arena->stats, &arena->decay_muzzy.stats.nmadvise);
+	astats->decay_muzzy.purged += arena_stats_read_u64(tsdn, &arena->stats,
+	    &arena->decay_muzzy.stats.purged);
+
 	astats->base += base_allocated;
 	astats->internal += arena_internal_get(arena);
 	astats->resident += base_resident + (((atomic_read_zu(&arena->nactive) +
-	    extents_npages_get(&arena->extents_cached)) << LG_PAGE));
+	    extents_npages_get(&arena->extents_dirty)) << LG_PAGE));
```

Comment:
This is actually missing ```+ extents_npages_get(&arena->extents_muzzy)```, which I happened to notice during a rebase (not pushed yet).